week6_04.py

```diff
@@ -0,0 +1,21 @@
+from PIL import Image
+
+
+def get_an_encrypted_message_from_the_image(image_path: str) -> str:
+    """
+    function gets a path to file and returns the message encrypted within it by the position of black pixels.
+    :param image_path:The path of the encryption image.
+    :return:The encrypted message.
+    """
+    the_message = ''
```

Comment:
Use double quotes on strings - 
https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python